app/javascript/mastodon/features/status/containers/detailed_status_container.js

```diff
@@ -50,20 +41,13 @@ const makeMapStateToProps = () => {
   return mapStateToProps;
 };
 
-const mapDispatchToProps = (dispatch, { intl }) => ({
+const mapDispatchToProps = (dispatch) => ({
 
   onReply (status) {
     dispatch((_, getState) => {
       let state = getState();
       if (state.getIn(['compose', 'text']).trim().length !== 0) {
-        dispatch(openModal({
-          modalType: 'CONFIRM',
-          modalProps: {
-            message: intl.formatMessage(messages.replyMessage),
-            confirm: intl.formatMessage(messages.replyConfirm),
-            onConfirm: () => dispatch(replyCompose(status)),
-          },
-        }));
+        dispatch(openModal({ type: 'CONFIRM_REPLY', status }));
```

Comment:
Shouldn't this be?
```suggestion
        dispatch(openModal({ type: 'CONFIRM_REPLY', modalProps: { status } }));
```

But looking more at the file I think the whole file may be dead code?

Reply:
Pushed a fix, but yes this seems to be dead code.

I can make another PR once this one is merged to remove it.